secretmanager/runtime/src/main/java/io/quarkiverse/googlecloudservices/secretmanager/runtime/SecretManagerCredentialsProvider.java

```diff
@@ -0,0 +1,39 @@
+package io.quarkiverse.googlecloudservices.secretmanager.runtime;
+
+import java.io.IOException;
+import java.util.Collections;
+import java.util.Map;
+
+import javax.inject.Inject;
+
+import com.google.cloud.secretmanager.v1.AccessSecretVersionResponse;
+import com.google.cloud.secretmanager.v1.SecretManagerServiceClient;
+import com.google.cloud.secretmanager.v1.SecretVersionName;
+
+import io.quarkiverse.googlecloudservices.common.GcpConfiguration;
+import io.quarkus.credentials.CredentialsProvider;
+
+public class SecretManagerCredentialsProvider implements CredentialsProvider {
```

Comment:
What is this SecretManagerCredentialsProvider used for here? Is it just an example use case for the secret manager setup?